src/components/ambassadorCard/ambassadorCard.module.scss

```diff
@@ -12,111 +12,99 @@
   justify-content: flex-start;
   align-items: flex-start;
   background-color: $accent-color;
-  border: 0.5rem solid $primary-color;
-  border-radius: 1rem;
   box-shadow: $shadow;
-  padding: 0.25rem 0 0.5rem;
+
+  border-radius: 1rem;
+  overflow: hidden;
 
   color: $primary-text;
   font-size: 0.7rem;
 
-  h3 {
-    font-size: 0.9rem;
-  }
-
-  .close {
-    position: absolute;
-    top: 0.25rem;
-    right: 0.25rem;
-    width: 2rem;
-    display: block;
-    font-size: 1.5rem;
-    line-height: 1.1;
-    cursor: pointer;
-    text-align: center;
-    transition: $transition;
-    transition-property: filter;
-
-    &:hover {
-      filter: brightness(0.7);
+  &.birthday {
+    &::after {
+      content: url("../../assets/party.png");
+      width: 4rem;
+      height: auto;
+      position: absolute;
+      left: 50%;
+      top: 0;
+      transform: translate(-50%, -90%);
     }
   }
 
-  .img {
+  .hero {
+    position: relative;
     width: 100%;
-    aspect-ratio: 3;
-    object-fit: cover;
-    align-self: center;
-    border-radius: 0;
-    margin-bottom: 0.25rem;
 
-    @media (min-width: 480px) {
+    .img {
+      display: block;
+      width: 100%;
+      object-fit: cover;
       aspect-ratio: 2.2;
-    }
-  }
-
-  .name {
-    font-size: 1.3rem;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
-    max-width: calc(100% - 3rem);
-    width: 100%;
-    flex-shrink: 0;
-  }
 
-  .name,
-  .row {
-    margin: 0 0.5rem 0.25rem;
-  }
-
-  .info {
-    display: inline-flex;
-    align-items: center;
-    gap: 0.5rem;
-  }
-
-  .compact {
-    display: flex;
-    flex-wrap: wrap;
-    gap: 0.25rem 1.5rem;
-  }
-
-  .mod {
-    display: flex;
-    gap: 0.5rem;
-    align-items: center;
+      @media (min-width: 480px) {
+        aspect-ratio: 1.8;
+      }
 
-    img {
-      width: 1.5rem;
-      height: 1.5rem;
-      object-fit: cover;
-      border-radius: 0;
+      &:hover {
+        + .overlay {
+          opacity: 0.125;
+          backdrop-filter: blur(0.125rem);
+        }
+      }
     }
-  }
-
-  .site {
-    font-style: italic;
-    margin-top: 0.75rem;
 
-    a {
-      color: $secondary-color;
-      text-decoration: underline;
-      white-space: nowrap;
+    .overlay {
+      display: flex;
+      position: absolute;
+      top: 0;
+      width: 100%;
+      height: 2.25rem;
+      background: rgba($accent-color, 0.5);
+      backdrop-filter: blur(0.25rem);
       transition: $transition;
-      transition-property: color;
+      transition-property: opacity, backdrop-filter;
+
+      .name {
+        font-size: 1.3rem;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        overflow: hidden;
+        padding: 0 0.5rem;
```

Comment:
```suggestion
        padding: 0.2rem 0.5rem 0;
```

Reply:
Doing this seems wrong to me? The name then becomes no longer vertically centered in the blur area?

Reply:
for me it does visually. the font sits a bit high in the line

Reply:
Ah yeah, I see now, does make it look more aligned to an imaginary baseline